assets/src/edit-story/components/checklist/checks/storyAmpValidationErrors.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { trackEvent } from '@web-stories-wp/tracking';
+import { useEffect, useRef, useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+import { ChecklistCard, DefaultFooterText } from '../../checklistCard';
+import { DISTRIBUTION_COPY } from '../constants';
+import { useRegisterCheck } from '../countContext';
+
+export async function getStoryAmpValidationErrors({ link, status }) {
+  if (!link || !['publish', 'future'].includes(status)) {
+    return false;
+  }
+
+  try {
+    const response = await fetch(link);
```

Comment:
Does this fetch the story markup that would ultimately be rendered? I.e. including all WP filters in PHP, etc.

Reply:
Yes